omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/check_token_using_endpoint.lua.erb

```diff
@@ -0,0 +1,24 @@
+ngx.req.read_body()
+local res
+
+if ngx.var.request_method == "POST" then
+  meth = ngx.HTTP_POST
+else
+  meth = ngx.HTTP_GET
```

Comment:
The idea is to be able to have a generic way of doing `/validate` that can handle other types of requests as well. This generic way is planned to be used by other features (e.g. compliance, etc..) in the future.